Makefile

```diff
@@ -285,6 +287,16 @@ generate-k8s-yaml:
 
 release: generate-k8s-yaml upload-resources-to-github
 
+setup-ec2-sdk-override:
+	@if [ "$(EC2_SDK_OVERRIDE)" = "y" ] ; then \
```

Comment:
I was trying to add similar to this - https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/appmesh_models_override/setup.sh and https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/Makefile#L96. So that if any new feature needs to generate the model, they can run the script. Maybe I will remove from make file and keep this script in a helper directory and provide comments on how to use it?

Reply:
> Maybe I will remove from make file and keep this script in a helper directory and provide comments on how to use it?

I like this plan.

Reply:
Thank you, will do that.